### PR TITLE
Build mysql container image. 

### PR DESCRIPTION
**Pull request by d-kuro**
refs: https://github.com/cybozu-go/moco/issues/535

Push mysqldcontainer image to ghcr.io.

This PR only pushes the image.
The pushed image needs to be changed to be used by MOCO, but this will be done in another PR.

### CODE REVIEW

File: .github/workflows/build-mysql-container.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                spec:
+                  accessModes: [ "ReadWriteOnce" ]
+                  resources:
+                    requests:
+                      storage: 1Gi
+          EOF
+          kubectl apply -f mycluster.yaml
+
+      - name: Wait for MySQLCluster
+        run: kubectl wait -n default --for=condition=Available mysqlcluster/test --timeout=180s
```
Comment (d-kuro) [inline .github/workflows/build-mysql-container.yaml:99]: Currently we are only testing until MySQLCluster is ready.
When the MySQL image is pushed to ghcr.io, we will rewrite the E2E manifests and modify it to run the E2E test using the built mysqld image.

https://github.com/cybozu-go/moco/blob/main/e2e/testdata/single.yaml#L26

File: containers/mysql/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# MySQL container
+
+This directory provides a Dockerfile to build a MySQL container for [MOCO](https://github.com/cybozu-go/moco).
+This also provides `moco-init` command to initialize MySQL data for MOCO.
```
Comment (masa213f) [inline containers/mysql/README.md:4]: From this update, mysql images will not contain moco-init.

File: containers/mysql/8.0.18/.dockerignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+*
+!workspace
```
Comment (masa213f) [inline containers/mysql/8.0.18/.dockerignore:2]: This exclusion is not necessary.

**Review by masa213f — CHANGES_REQUESTED** [review]

File: containers/mysql/8.0.18/.dockerignore (hunk 3 above)
Comment (d-kuro) [reply containers/mysql/8.0.18/.dockerignore]: Thanks, deleted.
https://github.com/cybozu-go/moco/pull/533/commits/2c2a3e4c5329364229e2ab5edf64c1dda7b65f1d

File: containers/mysql/README.md (hunk 2 above)
Comment (d-kuro) [reply containers/mysql/README.md]: Remove it: https://github.com/cybozu-go/moco/pull/533/commits/027acfe82595ec801aed515ffb96d87e4e938048

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**